src/styles/blocks/product.scss

```
@@ -0,0 +1,86 @@
+$main_color: #00acdc;
+$second_color: #616070;
+
+.product {
+  box-sizing: border-box;
+  width: 200px;
+
+  padding: 32px 16px 40px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    display: block;
+    margin: 0 auto ;
+    margin-bottom: 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin: 0;
+    margin-bottom: 4px;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin: 0;
+    margin-bottom: 16px;
+    color: $second_color;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    margin: 0;
+    margin-bottom: 24px;
+  }
+  &__price {
```

please separate different rules with empty lines
also check other cases
```suggestion
  }
  
  &__price {
```